src/pages/settings/Profile/PronounsPage.js

```diff
@@ -101,7 +110,7 @@ class PronounsPage extends Component {
      * @param {Object} selectedPronouns
      */
     updatePronouns(selectedPronouns) {
-        PersonalDetails.updatePronouns(selectedPronouns.value);
+        PersonalDetails.updatePronouns(selectedPronouns.keyForList === this.initiallyFocusedOption.keyForList ? '' : selectedPronouns.value);
```

Comment:
Just a minor change to ensure we're not getting the `value` property here:

```suggestion
        PersonalDetails.updatePronouns(selectedPronouns.keyForList === this.initiallyFocusedOption.keyForList ? '' : lodashGet(selectedPronouns, 'value', '');
```